R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2083,6 +2083,43 @@ test_that("dapply() on a DataFrame", {
   expect_identical(expected, result)
 })
 
+test_that("gapply() on a DataFrame", {
```

Comment:
You need to write a new test case for `gapply`. The test cases are completely different from `dapply`.